butterknife-compiler/src/main/java/butterknife/compiler/ButterKnifeProcessor.java

```diff
@@ -371,6 +397,33 @@
     return bindingMap;
   }
 
+  private BindingSet createStubBindingSet(TypeElement parentType) {
+    BindingSet parentBinding;
+    BindingSet.Builder parentBuilder = BindingSet.newBuilder(parentType);
+    if (hasViewBindings(parentType)) {
+      //add a fake field to the parent class so that it will indicate it has a view bindings.
+      //this is required for the subclass to generate a proper view binder
```

Comment:
https://nedbatchelder.com/blog/201401/comments_should_be_sentences.html

Reply:
I try to do that too. You mean it should start with upper case and end with a `.` ?

Reply:
yes